src/VisualStudio/VisualBasic/Impl/PackageRegistration.pkgdef

```diff
@@ -188,4 +188,11 @@
 "Indent Style"=dword:00000002
 
 [$RootKey$\VB Editor\Roslyn]
-"DisplayLineSeparators"=dword:00000001
\ No newline at end of file
+"DisplayLineSeparators"=dword:00000001
+
+// CacheTag value should be changed when registration file changes
+// See https://devdiv.visualstudio.com/DevDiv/_wiki/wikis/DevDiv.wiki/39345/Manifest-Build-Deployment-and-Setup-Authoring-In-Depth?anchor=example-pkgdef-key for more infomation
+[$RootKey$\SettingsManifests\{574fc912-f74f-4b4e-92c3-f695c208a2bb}]
+@="Microsoft.VisualStudio.LanguageServices.VisualBasic.VisualBasicPackage"
+"ManifestPath"="$PackageFolder$\UnifiedSettings\visualBasicSettings.registration.json"
+"CacheTag"=qword:08DC1824DFE0117B
```

Comment:
Not ideal, there are some ways to auto-generate this but it's another large change. So use this for the time being.

Reply:
Can we make the CacheTag be a xxHash64 of the content of **visualBasicSettings.registration.json** (normalize line endings to `\n` for the hash), and enforce this value in CI?

Reply:
I use xxHash128 and take its first 8 bytes to generate the value instead. 
Seems like xxHash64 is not avaible in the repo now. 
According to GPT it still has a low collision rate. So it should still fit our testing purpose here.